src/molecules/list/documentation.stories.mdx

```diff
@@ -62,6 +62,16 @@ export const Template = (args) => {
   </Story>
 </Canvas>
 
+<Canvas>
+  <Story name='Disabled List' args={{}}>
+    <List >
```

Comment:
theres an extra space here

Reply:
in fact this is bad because you're testing no-clickability but that happens by default when you dont have an onClickListener. You should have one in this example